src/clothing_mod.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#ifndef CLOTHING_MOD_H
+#define CLOTHING_MOD_H
+
+#include <stddef.h>
```

Comment:
```suggestion
#include <cstddef>
```
"stddef.h" is a C header, "cstddef" is the matching C++ header.